src/pages/portfolio/edit/PortfolioEditPage.tsx

```diff
@@ -258,6 +263,10 @@ const PortfolioEditPage = () => {
 		if (event.key === 'Tab') event.preventDefault();
 	};
 
+	if (!isLoading && !portfolio?.isWriter) {
+		return <NotFound />;
+	}
+
```

Comment:
```suggestion
	if (isSuccessReadPortfolio && portfolioId && !portfolio?.isWriter) {
		return <NotFound />;
	}

```

해당 부분을 이렇게 변경해야 편집뿐만아니라 작성도 가능합니다..! 안그러면 404가 뜨더라구요ㅠㅠ

Reply:
오오!! 적용이 된건가요??!!
감사합니다!! 저도 발견했었는데, 수정하는거를 깜빡했어요..ㅠ

Reply:
아니요! 해당 부분 이렇게 적용해주시면 감사할 것 같아요!!👍🏻